alicloud/resource_alicloud_cms_alarm_test.go

```diff
@@ -147,6 +147,7 @@ func TestAccAlicloudCmsAlarm_update(t *testing.T) {
 					resource.TestCheckResourceAttr("alicloud_cms_alarm.update", "name", "tf-testAccCmsAlarm_update"),
 					resource.TestCheckResourceAttr("alicloud_cms_alarm.update", "operator", "<="),
 					resource.TestCheckResourceAttr("alicloud_cms_alarm.update", "triggered_count", "2"),
+					resource.TestCheckResourceAttr("alicloud_cms_alarm.update", "webhook", "https://5167308351401688.eu-central-1.fc.aliyuncs.com/2016-08-15/proxy/Terraform/AlarmEndpointMock/"),
```

Comment:
I think this check using `TestMatchResourceAttr` will be better and it can hide you account id.

Reply:
done